dependencies/common/install-pandoc

```diff
@@ -20,6 +20,11 @@ set -e
 source "$(dirname "${BASH_SOURCE[0]}")/../tools/rstudio-tools.sh"
 section "Installing Pandoc"
 
+if [ "$(arch)" = "aarch64" ]; then
+  echo "Pandoc installer not yet available for aarch64"
```

Comment:
I believe it is (as of very recently): https://github.com/jgm/pandoc/releases/tag/2.16.1